features/sidebar.feature

```diff
@@ -53,5 +53,3 @@ Feature: Sidebar navigation
     Given I am on the "Show" page for project "hello mars"
     Then I should see "hello moon" before "hello pluto"
     And I should see "hello pluto" before "hello world"
-
-
```

Comment:
This whole feature :point_up: is particularly verbose. Reading the feature description and especially the scenario, it is not clear to me what the problem actually is. Don't write cucumber scenarios in a way that let's you re-use existing step definitions. 1) write your feature 2) run `bundle exec cucumber` 3) copy-paste the generated step definitions and implement them. It is OK to have many step defintions.

See this blog post for details: http://aslakhellesoy.com/post/11055981222/the-training-wheels-came-off
Aslak Hellesøy is the inventor of cucumber-ruby and he gives advice on how not to write boring scenarios. :wink:

Reply:
and there is more: https://cucumber.io/blog/2016/07/01/cucumber-antipatterns-part-one

Reply:
definite need for refactoring - well flagged @roschaefer 
@mattwr18 - yes definitely a great thing for a refactoring ticket (when you didn't write the original tests) - did you create a refactoring ticket already?